services/src/main/java/org/fao/geonet/api/records/MetadataWorkflowApi.java

```diff
@@ -390,6 +391,7 @@ public void deleteAllRecordStatus(
             @ApiParam(value = "End date", required = false) @RequestParam(required = false) String dateTo,
             @ApiParam(value = "From page", required = false) @RequestParam(required = false, defaultValue = "0") Integer from,
             @ApiParam(value = "Number of records to return", required = false) @RequestParam(required = false, defaultValue = "100") Integer size,
+            @ApiParam(value = "One or more status id. Default is all.", required = false) @RequestParam(required = false) List<String> statusIds,
```

Comment:
I believe this would be better if all the arrays are in the same location.  So maybe it should be moved this after "uuid" 
Also instead of `List<String> statusIds` - why not used `Integer[] statusId` to be consistent with other fields.